packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -18,10 +18,9 @@ import '_isolates_io.dart'
 typedef ComputeCallback<Q, R> = FutureOr<R> Function(Q message);
 
 /// The signature of [compute].
-typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q message, { String? debugLabel });
-
-/// Spawn an isolate, run `callback` on that isolate, passing it `message`, and
-/// (eventually) return the value returned by `callback`.
+///
+/// [compute] spawns an isolate, runs `callback` on that isolate, passes it
```

Comment:
nit: Sentences should start grammatically correct with a capital letter.

Reply:
I've tweaked the wording to avoid this.